### DISCUSSION TITLE
CPO pole figure

### DISCUSSION THREAD

**Discussion by EmilyColeur** [mtex-toolbox/mtex#1932] [Ask Anything]
Hello everyone! I'm a student and have just started working with ebsd, but I can't understand the pole figures. My aim is to understand how crystals are oriented through the interpretation of pole figure maps. In the image below is an example, how do I understand how the planes are oriented? Do you have any books to recommend me to study these figures in practice? I have tried reading texts but they do not explain how to read these maps.... 
Thank you very much!
Emily
![Catebsd](https://github.com/mtex-toolbox/mtex/assets/143271417/d69741b2-b410-4ee0-a110-6a2d9acc216b)

**Comment by junspark** [comment]
I highly recommend
Introduction to Texture Analysis
Macrotexture, Microtexture, and Orientation Mapping, Second Edition
by engler and randle

https://www.routledge.com/Introduction-to-Texture-Analysis-Macrotexture-Microtexture-and-Orientation/Engler-Randle/p/book/9781420063653?utm_source=cjaffiliates&utm_medium=affiliates&cjevent=df3e3af7442e11ee82ce2ba10a82b820

- jsp
________________________________________
From: EmilyColeur ***@***.***>
Sent: Saturday, August 26, 2023 9:04 AM
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] CPO pole figure (Discussion #1932)

Hello everyone! I'm a student and have just started working with ebsd, but I can't understand the pole figures. My aim is to understand how crystals are oriented through the interpretation of pole figure maps. In the image below is an example, how do I understand how the planes are oriented? Do you have any books to recommend me to study these figures in practice? I have tried reading texts but they do not explain how to read these maps....
Thank you very much!
Emily
[Catebsd]<https://user-images.githubusercontent.com/143271417/263468854-d69741b2-b410-4ee0-a110-6a2d9acc216b.JPG>

**Reply by ralfHielscher** [reply]
I can also recommend the next MTEX workshop https://mtex-toolbox.github.io/workshop24 :smile: 

Ralf.
